examples/meta/src/statistical_testing/linear_time_maximum_mean_discrepancy.sg

```diff
@@ -49,7 +49,7 @@ real width = learnt_kernel_single.get_real("log_width")
 #![select_kernel_combined]
 mmd.set_kernel_selection_strategy(enum EKernelSelectionMethod.KSM_MAXIMIZE_POWER, True)
 mmd.select_kernel()
-CombinedKernel learnt_kernel_combined = CombinedKernel:obtain_from_generic(mmd.get_kernel())
+CombinedKernel learnt_kernel_combined = mmd.get_kernel()
```

Comment:
.get_real_vector(“xxx”)
Where you need to check for xxx for the kernel weights are registered in CCombinedKernel.cpp (search for SG_ADD)